### DISCUSSION TITLE
Logstash missing on fresh install. Twice in a row

### DISCUSSION THREAD

**Discussion by dschmitz05** [Security-Onion-Solutions/securityonion#11926] [2.4] [answered]
### Version

2.4.3

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

32

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Logstash missing, have tried so-logstash-restart --force hoping it would reinstall. Still comes up with it missing. Have tried fresh install twice. There are no logs in /opt/so/log/logstash

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
You specify above that your version is 2.4.3. Is that correct? That was an old Beta version.

You also specify that the installation type was Standalone. On the installation type screen, did you select Standalone or did you perhaps select Eval?
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/0cd59f9d-a69b-47e9-8a19-6cf8ab178e5e)

If you don't remember what you picked, you can run the following command:
```
sudo grep "Node Type" /root/sosetup.log
```

If you chose Eval, then please note that Evaluation Mode does not run Logstash at all:
https://docs.securityonion.net/en/2.4/architecture.html#evaluation

**Comment by dschmitz05** [comment]
I just verified that my installation type is Standalone. As for the version of install, it was the most recent verision available for download yesterday. I compared the SHA256 checksum and it came back correct.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by dougburks** [comment]
What is the output of the following?
```
sudo docker images |grep logstash
```

**Comment by dschmitz05** [comment]
There is no output when I run that command.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by dougburks** [comment]
Were there any errors when you ran Setup?

Are you able to share /root/sosetup.log?

**Comment by dschmitz05** [comment, marked as answer]
I caved and reuploaded my ISO file to my ESXi and tried reinstalling for the 3rd time. This time there were no issues. I apologize for the inconvenience.


Thank you!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
